packages/entity-browser/src/dev/fetchMocks.js

```diff
@@ -1,10 +1,39 @@
 import {utilFetchMocks} from 'tocco-util/dev'
 
+import {createUsers} from './recordFactory'
+
+const allRecords = createUsers(1000)
+
 export default function setupFetchMock(fetchMock) {
   utilFetchMocks.sessionFetchMock(fetchMock)
   utilFetchMocks.textResourceFetchMock(fetchMock, require('./messages.json'))
-  
-  fetchMock.get(new RegExp('^.*?/nice2/rest/entities/User$'), require('./users.json'))
+  fetchMock.get(new RegExp('^.*?/nice2/rest/form/.*?'), require('./user_list.json'))
+
+  fetchMock.get(new RegExp('^.*?/nice2/rest/entities/.*/count?'), {'count': 1000})
+  fetchMock.get(new RegExp('^.*?/nice2/rest/entities/.*?'), (url, opts) => {
```

Comment:
what is the purpose of the question marks at the end of the expressions on line 10, 12 and 13?